src/routes/home/index.js

```diff
@@ -1,10 +1,23 @@
 import { h } from 'preact';
-import style from './style.scss';
+import Button from '../../components/Button';
+// import style from './style.scss';
```

Comment:
Remove console.logs 🐵

Reply:
U mean unused comment for style? Cuz there's no console.log here :>

Reply:
My bad :D